sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/parallel_func.py

```diff
@@ -99,80 +99,96 @@ def parallel_run_function(
-        through PipelineParameter.
-    :type mini_batch_size: str
-    :keyword partition_keys: The keys used to partition dataset into mini-batches.
-        If specified, the data with the same key will be partitioned into the same mini-batch.
-        If both partition_keys and mini_batch_size are specified, the partition keys will take effect.
-        The input(s) must be partitioned dataset(s),
-        and the partition_keys must be a subset of the keys of every input dataset for this to work.
-    :type partition_keys: List
-    :keyword input_data: The input data.
-    :type input_data: str
-    :keyword inputs: a dict of inputs used by this parallel.
-    :type inputs: Dict
-    :keyword outputs: the outputs of this parallel
-    :type outputs: Dict
-    :keyword instance_count: Optional number of instances or nodes used by the compute target. Defaults to 1.
-    :type instance_count: int
-    :keyword instance_type: Optional type of VM used as supported by the compute target.
-    :type instance_type: str
-    :keyword docker_args: Extra arguments to pass to the Docker run command. This would override any
-     parameters that have already been set by the system, or in this section. This parameter is only
-     supported for Azure ML compute types.
-    :type docker_args: str
-    :keyword shm_size: Size of the docker container's shared memory block. This should be in the
-     format of (number)(unit) where number as to be greater than 0 and the unit can be one of
-     b(bytes), k(kilobytes), m(megabytes), or g(gigabytes).
-    :type shm_size: str
-    :keyword identity: Identity that training job will use while running on compute.
-    :type identity: Union[ManagedIdentity, AmlToken]
-    :keyword is_deterministic: Specify whether the parallel will return same output given same input.
-        If a parallel (component) is deterministic, when use it as a node/step in a pipeline,
-        it will reuse results from a previous submitted job in current workspace which has same inputs and settings.
-        In this case, this step will not use any compute resource.
-        Default to be True, specify is_deterministic=False if you would like to avoid such reuse behavior.
+    :param name: Name of the parallel job or component created, defaults to None.
+    :type name: str, optional
+    :param description: A friendly description of the parallel, defaults to None.
+    :type description: str, optional
+    :param tags: Tags to be attached to this parallel, defaults to None.
+    :type tags: Dict, optional
+    :param properties: The asset property dictionary, defaults to None.
+    :type properties: Dict, optional
+    :param display_name: A friendly name, defaults to None.
+    :type display_name: str, optional
+    :param experiment_name: Name of the experiment the job will be created under,
+                            if None is provided, default will be set to current directory name.
```

Comment:
A bit confused that for some lines you made the indent to be only 4-char long, but here you make indent to be more than 4-chars, which one is the preferred?

Reply:
There is no fixed indentation, just to align the previous line.
Like 'if None .....' align the 'Name of the ....'